test/metabase/api/user_test.clj

```diff
@@ -1252,16 +1252,14 @@
 
 (deftest user-update-event-test
   (testing "User Updates via the API are recorded in the audit log"
-    (mt/with-model-cleanup [:model/Activity :model/AuditLog]
-      (t2.with-temp/with-temp [User {:keys [id]} {:first_name "John"
-                                                  :last_name  "Cena"}]
-          (testing "PUT /api/user/:id"
-            (mt/user-http-request :crowberto :put 200 (format "user/%s" id)
-                                  {:first_name "Johnny" :last_name "Appleseed"})
-            (is (= {:topic    :user-update
-                    :user_id  (mt/user->id :crowberto)
-                    :model    "User"
-                    :model_id id
-                    :details  {:first_name "Johnny"
-                               :last_name "Appleseed"}}
-                   (audit-log-test/event :user-update id))))))))
+    (t2.with-temp/with-temp [User {:keys [id]} {:first_name "John" :last_name  "Cena"}]
+      (testing "PUT /api/user/:id"
+        (mt/user-http-request :crowberto :put 200 (format "user/%s" id)
+                              {:first_name "Johnny" :last_name "Appleseed"})
+        (is (= {:topic    :user-update
```

Comment:
I was using `=?` because some audit_log details have timestamps in them. So I was using them to just check that the timestamps were there, not that they were equal. Since this is straight up map equality `=` works.

aside: I think `=?` would also work, but best to save it for when it's needed so it's not confusing.